decidim-core/lib/decidim/api/functions/user_entity_finder.rb

```diff
@@ -23,6 +23,7 @@ def call(_obj, args, ctx)
         end
         Decidim::UserBaseEntity
           .where.not(confirmed_at: nil)
+          .where(blocked_at: nil)
```

Comment:
Maybe would make any sense not to include the deleded users ( groups ) in the search results ?

Also, Would make any sense to use scopes in the search ? 
I am thinking that we may move in the super class: 

https://github.com/decidim/decidim/blob/e4a382edf15222845d7f561292c77e00682c56e1/decidim-core/app/models/decidim/user.rb#L66-L70

Reply:
I'm not sure about the deleted users part, what do @decidim/product think of this?

Regarding the use of scopes, it makes sense but this is outside the scope of this P.R. and I think we should work on this in another one.

Reply:
Also, I'm not sure about the usage of the scope, UserBaseEntity doesn't respond to these scopes which are only implemented on User class afaik.

Reply:
> I'm not sure about the deleted users part, what do @decidim/product think of this?

Deleted or blocked user shouldn't be visible in search/mentions/etc

Reply:
> I guess deleted or blocked User Groups should not be visible either ...
>
> @decidim/product, what do you think about it?

Maybe I didn't understand the question, I'd rephrase it: deleted or hidden (moderated) Users or User Groups shouldn't be visible in search/mentions/etc. They should have the same behavior in both cases, I can't think of any place where we would want a non-consistent between deleted/hidden Users and User Groups - can you think of any? Is there something that I'm missing?